tryutmpx.c

```diff
@@ -0,0 +1,6 @@
+#include <utmpx.h>
+
+void main()
+{
+  ;
```

Comment:
this should probably also include the usage of struct utmpx, ideally probing those members that are actually used in qbiff (i.e. ut_line, ut_type, and ut_user).